release-calendar/src/types.ts

```diff
@@ -55,6 +53,6 @@ export class Promotion {
 }
 
 export interface ApiServiceInterface {
-  getReleases: () => Promise<ReleaseViewModel[]>;
+  getReleases: () => Promise<EventInput[]>;
```

Comment:
The api service uses `EventInput` from `view-models` but the interface uses fullcalendar's? Not sure what's going on.

Reply:
On that note why don't we get rid of this interface. It's not really helping!